app/scripts/modules/core/src/cluster/ClusterPod.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import autoBindMethods from 'class-autobind-decorator';
+
+import { NgReact, ReactInjector } from 'core/reactShims';
+import { ServerGroup } from 'core/serverGroup/ServerGroup';
+import { Application } from 'core/application';
+import { EntityNotifications } from 'core/entityTag/notifications/EntityNotifications';
+import { HealthCounts } from 'core/healthCounts';
+import { IServerGroup } from 'core/domain';
+import { Tooltip } from 'core/presentation';
+import { IClusterSubgroup, IServerGroupSubgroup } from './filter/clusterFilter.service';
+
+export interface IClusterPodProps {
+  grouping: IClusterSubgroup;
+  application: Application;
+  parentHeading: string;
+  sortFilter: any;
+}
+
+export interface IClusterPodState {
+  showCloseButton: boolean;
+}
+
+@autoBindMethods
+export class ClusterPod extends React.Component<IClusterPodProps, IClusterPodState> {
+  constructor(props: IClusterPodProps) {
+    super(props);
+
+    this.state = {
+      showCloseButton: props.application.getDataSource('serverGroups').fetchOnDemand === true,
+    };
+  }
+
+  public close(): void {
+    const { clusterFilterModel } = ReactInjector;
+    const { parentHeading, grouping, application } = this.props;
+
+    delete clusterFilterModel.asFilterModel.sortFilter.clusters[`${parentHeading}:${grouping.heading}`];
```

Comment:
this is actually cluster filtering for when #clusters > 250 (`ksrouter.euwest1.ural`) .  It's not expanded/collapsed state